gef.py

```diff
@@ -10382,6 +10382,14 @@ def __gef_prompt__(current_prompt):
             site.addsitedir(site_packages_dir)
         except FileNotFoundError:
             pass
+        
+        # when Python virtual enviromrnt is used GDB still loads system Python
+        # so GEF doesn't load site-packages dir from environment
+        pythonbin = which("python3")
+        PREFIX = gef_pystring(subprocess.check_output([pythonbin, '-c', 'import os,sys;print((sys.prefix))'])).strip("\\n")
```

Comment:
Why `strip("\\n")`?

Reply:
`sys.base_prefix` comes without `\\n`. Without it I can't validate if GEF is run inside Venv .